test/load/traders/random-trader.js

```diff
@@ -0,0 +1,33 @@
+const Promish = require('promish');
+const Trader = require('./trader');
+
+class RandomTrader extends Trader {
+  start() {
```

Comment:
The intention for RandomTrader is that it will start with a configured market price and then "drift" away from that randomly.
So it should start by setting this.marketPrice = this.config.price
Then, each iteration, it will add or subtract a random amount multiplied by a configurable setting

Also - when trades happen, the trader should reset its own marketPrice to equal the trade price. This will keep the swarm of traders fairly close to each other. To implement this, you will need to modify the Trader base class to call an optional function like 'processTrade' whenever trades appear on the web-socket -> then in RandomTrader, you can implement processTrade to set the marketPrice